.github/workflows/test.yml

```
@@ -76,6 +76,9 @@ jobs:
     name: Merge Group
     needs: [prepare]
     uses: ./.github/workflows/merge-group.yml
+    # Those are not actually used on merge_group, but will throw an error if not set.
```

I assume you meant "on `merge-group.yml`" rather then "on `merge_group`", but it'd probably be clearer to say:

```suggestion
    # Those are not actually used when testing, but will throw an error if not set.
```

Or:


```suggestion
    # Those are not actually used on pull_request, but will throw an error if not set.
```